pages/metrics.tsx

```diff
@@ -0,0 +1,38 @@
+import debugFactory from 'debug'
+import React, { useEffect, useState } from 'react'
+import Container from 'semantic-ui-react/dist/commonjs/elements/Container'
+
+import MetricsApi from '@/api/MetricsApi'
+
+import ErrorsBox from '@/components/ErrorsBox'
+import Layout from '@/components/Layout'
+import MetricsTable from '@/components/MetricsTable'
+
+import { MetricBare } from '@/models/index'
+
+const debug = debugFactory('abacus:pages/metrics.tsx')
+
+const MetricsPage = function MetricsPage() {
+  debug('MetricsPage#render')
+  const [error, setError] = useState<Error | null>(null)
+  const [metrics, setMetrics] = useState<MetricBare[] | null>(null)
+
+  useEffect(() => {
+    MetricsApi.findAll()
+      .then((metrics) => setMetrics(metrics))
+      .catch(setError)
+  }, [])
```

Comment:
Can be simplified using `async`:

```suggestion
  useEffect(async () => {
    try {
      setMetrics(await MetricsApi.findAll())
    } catch (error) {
      setError(error)
    }
  }, [])
```

Reply:
How is this simpler? :slightly_smiling_face: 

The suggestion is longer and more syntax-heavy &ndash; I find it less readable.